### ISSUE TITLE
GetAllBlooks doesn't work. When u refresh the blooks disappears

### ISSUE THREAD

**Issue by MiliyZ** [Jude-Gideon/Blooket#49]
As title

**Comment by Jude-Gideon** [comment]
Yeah I know, you can't keep all Blooks forever, it's a temporary hack.
![image](https://user-images.githubusercontent.com/113489420/202879606-2b5806cd-2e78-447c-87da-636725d3a13e.png)
